tests/unit/functions/github.test.ts

```
@@ -153,14 +179,24 @@ describe("createGithubTeam", () => {
   });
 
   it("should rethrow BaseError instances", async () => {
-    const baseError = new GithubError({ message: "Failed to create GitHub team." });
-    mockOctokit.request.mockRejectedValueOnce(baseError);
+    const baseError = new GithubError({
+      message: "Failed to create GitHub team.",
+    });
+    mockOctokit.paginate.iterator.mockReturnValue(
+      (async function* () {
+        throw baseError;
+      })()
```

ESLint

[error] This generator function does not have 'yield'.

(require-yield)